params/network_upgrades.go

```diff
@@ -177,6 +184,12 @@ func (n *NetworkUpgrades) IsEtna(time uint64) bool {
 	return isTimestampForked(n.EtnaTimestamp, time)
 }
 
+// IsFUpgrade returns whether [time] represents a block
```

Comment:
brackets won't work in that case, use backtickets or nothing to not mislead the docs reader / dev trying to right click it.

```suggestion
// IsFUpgrade returns whether `time` represents a block
```

Reply:
I think it probably makes sense to fix `IsFUpgrade`, `IsEtna`, ect all in the same PR. After talking with @darioush it seems like this makes the most sense to do after the libevm migration.